pulp_container/app/serializers.py

```diff
@@ -774,11 +773,10 @@ class OCIBuildImageSerializer(ValidateFieldsMixin, serializers.Serializer):
     tag = serializers.CharField(
```

Comment:
Sorry, i cannot comment on 10 lines above ...
I think, we should not keep the `containerfile_artifact` as is.

Reply:
Hum... I could think of 2 ways to do that:
- provide a new field to pass the pulp_href of the File Content with the Containerfile (not sure if this could be considered an advantage, but with this approach users could use a different repository to store the Containerfile or pass the same as the build_context)  
- use the build_context field and add a logic to validate if the Containerfile is present in the specified File Repository

What do you think of these ideas?

Reply:
I think it's valuable to be able to use the same build-context with different Container files.

That being said, we could add a containerfile name field to select one file from the build context. Or, yes, we just allow to specify a file_content item for now. I don't know exactly what workflows we need to cover. We still (already) have the option to upload a file on the go, right? So adding more options feels not too bad.